sway/tree/view.c

```diff
@@ -88,6 +88,9 @@ void view_autoconfigure(struct sway_view *view) {
 	}
 
 	double x, y, width, height;
+
```

Comment:
Is it possible to remove the newlines here and after the assignment?

Reply:
Done!